multimodal/src/autogluon/multimodal/optimization/lit_mmdet.py

```diff
@@ -88,6 +88,13 @@ def _get_map_input(self, pred_results, sample):
             boxes = []
             scores = []
             labels = []
+            if isinstance(img_result[0], list):
+                # has one additional dimention with 2 outputs: img_result=img_result[0], mask_result=img_result[1]
+                raise NotImplementedError(
+                    "Do not support training for models with masks like mask r-cnn, "
```

Comment:
So for all models with mask branch, we can't train the model? Or we just don't have results for mask rcnn like model?

Reply:
Currently we can't train mask-rcnn, cascade mask-rcnn, and htc.